index/index.py

```diff
@@ -0,0 +1,69 @@
+import json
+import cache
+import os
+import asyncio
+
+dynamodb = None
+scheduler = None
+
+loop = asyncio.get_event_loop()
+
+def lambda_handler(event,context):
+    return loop.run_until_complete(job(event,context))
+
+async def job(event, context):
+    dynamodb,scheduler = cache.exports()
+
+    params = event.body
```

Comment:
Note that if you're executing a test event from the Lambda console, you would use just `event`. If the Lambda receives stringified JSON from an HTTP request, then you would need to use `json.loads(event.body)`